src/test/java/uk/gov/companieshouse/acsp/manage/users/integration/UserAcspMembershipIntegrationTest.java

```diff
@@ -710,6 +720,882 @@ void addAcspMemberSuccessWhenInviteeHasRemovedMembershipInDifferentAcsp() throws
     }
   }
 
+  @Test
```

Comment:
Could we maybe wrap these tests in a nested class, to separate them from the `GET` and `POST` tests?  This is an existing pattern in the file:

<img width="834" alt="Screenshot 2024-07-09 at 11 10 02" src="https://github.com/companieshouse/acsp-manage-users-api/assets/170090135/68528c20-8661-4103-b211-5517608b4e86">

```java
@Nested
@DisplayName("PATCH /acsp-members Tests")
class PatchAcspMembershipTests {
...
```